orthanc/orthanc-anon/plugin/pixl.py

```diff
@@ -156,6 +157,34 @@ def SendViaStow(resourceId):
         orthanc.LogError("Failed to send via STOW")
 
 
+def SendViaFTPS(resourceId: str) -> None:
+    """
+    Makes a POST API call to upload the resource to a dicom-web server
+    using orthanc credentials as authorisation
+    """
+    ORTHANC_USERNAME = config("ORTHANC_USERNAME")
+    ORTHANC_PASSWORD = config("ORTHANC_PASSWORD")
+
+    orthanc_url = "http://localhost:8042"
+
+    # Query orthanc-anon for the study
+    query = f"{orthanc_url}/studies/{resourceId}/archive"
+    try:
+        response_study = requests.get(query, auth=(ORTHANC_USERNAME, ORTHANC_PASSWORD), timeout=10)
+        success_code = 200
+        if response_study.status_code != success_code:
+            msg = "Could not download archive of resource '%s'"
+            raise RuntimeError(msg, resourceId)
+    except requests.exceptions.RequestException:
+        orthanc.LogError(f"Failed to query'{resourceId}'")
+
+    # get the zip content
+    zip_content = response_study.content
+    logging.info("Downloaded data: %s", zip_content)
+
+    upload.upload_content(zip_content, resourceId)
```

Comment:
ooh I think this is the wrong function being called, should it not be `upload_dicom_image`

Reply:
Wouldn't the zip_content be a BinaryIO in this case so an extra argument of remote_dir would need to be added? Or need to pass in a file_path instead of zip_content so would need to download first

Reply:
Yeah good point, I think we can just merge `upload_dicom_image()` and `upload_content()`, because for the parquet files we'll need a different function anyway.